lineman/app/services/current_user.coffee

```diff
@@ -4,8 +4,15 @@ angular.module('loomioApp').factory 'CurrentUser', ($rootScope, Records) ->
     window.Loomio.seedRecords.users = [] unless window.Loomio.seedRecords.users?
     window.Loomio.seedRecords.users.push window.Loomio.seedRecords.current_user
     Records.import(window.Loomio.seedRecords)
+
+    currentUser =  Records.users.find(window.Loomio.currentUserId)
+
     Records.memberships.fetchMyMemberships().then ->
+      currentUser.updateFromJSON membershipsLoaded: true
       $rootScope.$broadcast 'currentUserMembershipsLoaded'
-    currentUser =  Records.users.find(window.Loomio.currentUserId)
+
+    Records.discussions.fetchInbox().then ->
```

Comment:
Here's a thing I might have thought was crazy before I did it...

Today, the unread count query happens in the navbar directive. That's weird. Next best place? Well, I picked the CurrentUser model, alongside the memberships query. (after all, we're _always_ performing this request when you're logged in, because we always want that unread count to show up immediately) 

I can't help but think there may be a separate place to put this user / app initialization stuff though; it's a bit split between app.coffee and CurrentUser now, and current user feels like it's doing maybe 3 things at the moment, which it should really only be doing `Records.users.find(Loomio.currentUserId)`